game_eggs/teeworlds/README.md

```diff
@@ -1,13 +1,6 @@
 # Teeworlds
 
-## From their [Site](https://www.teeworlds.com/)
```

Comment:
I would rather keep this line.

Reply:
Hey, thanks for looking into this!
The line you mentioned is still there, it just [moved to the teeworlds readme.md](https://github.com/parkervcp/eggs/pull/1848/files#diff-8605cb9c84faa44a47bca635af71bf4786b24d2a0f0e7f5def9ac5c6422d55d1). Git is diffing the new directory readme.md against the old structure where teeworlds had no further sub directories.